domain/src/main/java/model/Type.kt

```diff
@@ -0,0 +1,6 @@
+package model
+
+enum class Type(val type: String) {
+    Boolean("boolean"),
```

Comment:
It would be better to rename this entry, because its value matches kotlin type `Boolean`